app-shells/starship/starship-1.16.0.ebuild

```diff
@@ -0,0 +1,432 @@
+# Copyright 2022-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+CRATES="
```

Comment:
Please use newer syntax of CRATES, `name@version`, which results in:

```
CRATES="
    adler@1.0.2
    ...
"
```

For future bumps, you can use pycargoebuild which is the new best tool for rust ebuilds.